Add SRG SSR Maven repository to publish repository (#1009)
SRGSSR/pillarbox-android: docs/README.md

gpr.user=<your_GitHub_username>
gpr.key=<your_GitHub_personal_access_token>
```
> [!TIP]
> You can check the [GitHub documentation](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-gradle-registry#using-a-published-package) for more information.

> ### Add SRG SSR Maven repository

Reviewer: ```suggestion
### Add the SRG SSR Maven repository
```

> Pillarbox is also deployed to [SRG SSR Maven repository](https://nxrm.rts.ch/repository/maven-srgssr/). So you need to add the following repository in your Gradle configuration:

Reviewer: ```suggestion
Pillarbox is also deployed to the [SRG SSR Maven repository](https://nxrm.rts.ch/repository/maven-srgssr/). If you don't want to use the GitHub Packages repository, add the following to your Gradle configuration:

> [!NOTE]
> You only need to add either the GitHub Packages repository or the SRG SSR repository, not both.
```

```kotlin
// If you declare your repositories in the `settings.gradle(.kts)` file
repositories {
    maven("https://nxrm.rts.ch/repository/maven-srgssr/") {
        credentials {
            username = providers.gradleProperty("srg_maven_user").get()
            password = providers.gradleProperty("srg_maven_password").get()
        }
    }
}

// If you declare your repositories in the root `build.gradle(.kts)` file
repositories {
    maven("https://nxrm.rts.ch/repository/maven-srgssr/") {
        credentials {
            username = project.findProperty("srg_maven_user")?.toString()
            password = project.findProperty("srg_maven_password")?.toString()
        }
    }
}
```

> The credential information can be found into the SRG SSR Bitwarden.

Reviewer: ```suggestion
The necessary credentials can be found in the SRG SSR Bitwarden.
```
